scripts/publish.js

```diff
@@ -17,16 +17,31 @@ shell.set('+v');
  *
```

Comment:
We might want to remove [Line 4](https://github.com/forcedotcom/salesforcedx-vscode/pull/1905/files#diff-e6889d205eee7cf0d8b917c5576051c8L4) because when we run `shell.exec` and if it fails, it will automatically exit without providing the reason why and makes debugging difficult (ran into this problem with Jason). We already check the exitCode and if it's non-zero we do `process.exit`

Reply:
Cool, I'll add that to these changes